.gitignore

```diff
@@ -1,5 +1,3 @@
-tests/data-files/external
-
```

Comment:
Why was this directory removed? Generally the external data files _shouldn't_ be checked in.

Reply:
Ignorance. I thought I added it during some of my testing. Didn't remember that it was part of the template .gitignore. Will add back.